### DISCUSSION TITLE
Restoring boot_backup after second installation process

### DISCUSSION THREAD

**Discussion by howl** [dangowrt/owrt-ubi-installer#170] [Q&A] [answered]
With the conversion of fip and factory to ubi partitions will be needed to rerun the installation procedure so boot_backup will be overwritten.

Is possible to dump the boot_backup content for example with dd, do the installation procedure and then restore again with dd the content of the partition?

**Comment by dangowrt** [comment, marked as answer]
The best is to simply mount the volume and copy the files from within there off the device.
```
mount ubi0:boot_backup /mnt -t ubifs
```
Now you should be able to access the backup files in `/mnt` and copy them from the device using `scp`.

You can (and that's a good idea imho) then write those files back to the boot_backup partition in the same way after have re-run the installer.

**Comment by howl** [comment]
Thanks a lot for the response, if that was for some reason not possible I was thinking about restoring original firmware but prefer not to know if they have first MiBs with badblocks.

**Comment by howl** [comment]
I will better reopen this, as it remains "hidden" as closed and in the next months it could be useful for people, coming from previous installers.
